<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="radioBubble">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Bubble</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBinary">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>BinaryInsert</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioShell">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Shell</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioPyramid">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Pyramide</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Type of sort:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Type of data:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Numbers</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Results:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textInput">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>451</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textOutput">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>451</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Input data:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Output data:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDone">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Выполнить сортировку</string>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushLoad">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Загрузить данные для обработки</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textRearrange">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textCompare">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rearrange:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Compare:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
